docpages/building/archlinux.md

```diff
@@ -0,0 +1,53 @@
+\page build-archlinux Building on Arch Linux
+
+\note You might not have to build library from source on arch, you can install directly from AUR repo, See the guide [here](/install-arch-aur.html).
+
+## 1. Copy Source code
+```bash
+git clone https://github.com/brainboxdotcc/DPP
+cd DPP/
+```
+
+```bash
+cmake -B ./build -DCMAKE_INSTALL_PREFIX=/usr/
```

Comment:
Why the `-DCMAKE_INSTALL_PREFIX=/usr/` here? You've said below that `you should install in /usr/ because it is the default system header location on arch.`, so if that's the case, then this is going to happen from `make install` regardless?

Reply:
no it was not happing i checked it and it was going in /usr/local only by soing make install

Reply:
Ah rightio.